ui/src/app/settings/views/Images/ThirdPartyDriversForm/ThirdPartyDriversForm.test.tsx

```diff
@@ -28,17 +28,6 @@ describe("ThirdPartyDriversForm", () => {
     });
   });
 
-  it("can render", () => {
-    const store = mockStore(state);
-
-    const wrapper = shallow(
-      <Provider store={store}>
-        <ThirdPartyDriversForm />
-      </Provider>
-    );
-    expect(wrapper).toMatchSnapshot();
```

Comment:
These snapshots were capturing the entire state tree as well as the external Redux API. I've removed them entirely as I don't see a point to them.